frontend/src/addEventPage/ListUsers.js

```diff
@@ -1,7 +1,7 @@
 import React, { useContext } from 'react';
 import EventContext from '../contexts/EventContext';
 
-export default function ListUser() {
+export default function ListUsers() {
```

Comment:
Refering to line 10: Does your user model contain an id field? If so it would be better suited as a key prop, two people with the same name is not that unlikely.